### PR TITLE
Fix lint warnings

### PR DESCRIPTION
**Pull request by mpeyper**

### CODE REVIEW

File: src/Element.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ export interface Element {
   yearDiscovered: number;
   valency: string;
   wrongValences: string[];
-  realname: any;
```
Comment (mpeyper) [inline src/Element.ts:24]: this was unused as far as I could tell.

**Comment by MohitPopli** [comment]
Instead of using `?` can we explicitly add null checks or undefined checks?
We can also use some utility functions from lodash such as `isNil`.

Just a suggestion :)

**Comment by mpeyper** [comment]
> Instead of using `?` can we explicitly add null checks or undefined checks?
> We can also use some utility functions from lodash such as `isNil`.

If there a specific check you're worried about? I'm happy to update them if there is. Also happy if you want to make the changes off this branch and U can merge your commits in.

Personally, I prefer `ts-is-present` for that sort of thing, but if `lodash` is already being used then it may be easier to use that.

**Comment by JacobMGEvans** [comment]
> > Instead of using `?` can we explicitly add null checks or undefined checks?
> > We can also use some utility functions from lodash such as `isNil`.
> 
> If there a specific check you're worried about? I'm happy to update them if there is. Also happy if you want to make the changes off this branch and U can merge your commits in.
> 
> Personally, I prefer `ts-is-present` for that sort of thing, but if `lodash` is already being used then it may be easier to use that.

@mpeyper  @MohitPopli 
I will never condone using something like Lodash when native code will do. ESPECIALLY simple validation checks, in reality, it would likely just be a type guard since we are using TypeScript.

File: src/NativeBridge.ts [hunk 2]
```diff
@@ -21,11 +21,12 @@ class NativeBridge {
 
   private exec<T extends keyof typeof AtomNative>(
     methodName: T
-    // @ts-ignore fix this
-  ): ReturnType<typeof AtomNative[T]> {
+  ): ReturnType<NonNullable<typeof AtomNative[T]>> {
```
Comment (JacobMGEvans) [inline src/NativeBridge.ts:24]: Nice

**Review by JacobMGEvans — APPROVED** [review]

**Pull request merged**